### PR TITLE
Add DisableImplicitFrameworkReferences

### LINKED ISSUE #18946: Please document the DisableImplicitFrameworkReferences property
I couldn't remember what it was called. It took running `msbuild /pp` and poring over the output to spot it.

Same as https://github.com/dotnet/docs/issues/18945.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59d4dafc-2217-9a7b-4590-27faa85164cd
* Version Independent ID: 64b80e80-1c15-c293-466a-918de98e96f6
* Content: [MSBuild properties for Microsoft.NET.Sdk - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props)
* Content Source: [docs/core/project-sdk/msbuild-props.md](https://github.com/dotnet/docs/blob/master/docs/core/project-sdk/msbuild-props.md)
* Product: **dotnet-core**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #18946

### CODE REVIEW

File: docs/core/project-sdk/msbuild-props.md [hunk 1]
```diff
@@ -376,6 +377,16 @@ You can set the `AssetTargetFallback` property to one or more [target framework
 </PropertyGroup>
 ```
 
+### DisableImplicitFrameworkReferences
+
+The `DisableImplicitFrameworkReferences` property lets you control whether to implicitly reference metapackages, such as [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) or [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library). Metapackages, which are sets of packages that are meaningful together, are implicitly referenced based on the [TargetFramework](#targetframework). Set this property to `true` to disable implicit [PackageReference](#packagereference) items to packages in the metapackage. If you set this property to `true`, you can add explicit references to just the packages you need.
```
Comment (tdykstra) [inline docs/core/project-sdk/msbuild-props.md:382]:
```suggestion
The `DisableImplicitFrameworkReferences` property lets you control whether to implicitly reference metapackages, such as [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) or [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library). Metapackages, which are packages that consist only of dependencies on other packages, are implicitly referenced based on the [TargetFramework](#targetframework). Set this property to `true` to disable implicit [PackageReference](#packagereference) items to packages in the metapackage. If you set this property to `true`, you can add explicit references to just the packages you need.
```

**Review by tdykstra — APPROVED** [review]
LGTM, one suggestion.

**Comment by gewarren** [comment]
Thanks for popping in to review this Tom! I'll be OOF next week :D

**Comment by dsplaisted** [comment]
@gewarren We don't use metapackages anymore for newer frameworks.  This property also controls implicit `FrameworkReference` items in .NET Core 3 and higher, as well as some implicit references such as System and System.Core when targeting .NET Framework.

**Pull request merged**